### DISCUSSION TITLE
pcl2.5.4版本中咋退出正版登录

### DISCUSSION THREAD

**Discussion by Sqrt404** [Meloong-Git/PCL#1723] [提问] [answered]
pcl2.5.4版本中咋退出正版登录

**Comment by WForst-Breeze** [comment, marked as answer]
将鼠标悬浮至头像，头像下方会显示功能区，最右侧图标即为退出登录

![0f78c82bda47e5093559882da195728bf795888d0a98a5a6daba451b54a9e395.0.PNG](https://user-images.githubusercontent.com/110760354/235085327-103d534f-599c-4316-8863-c1412addf8ca.PNG)
